basxconnect/mailer_integration/layouts.py

```diff
@@ -51,19 +54,32 @@ def _display_preferences(email):
             C(
                 hg.SPAN(email.email, style="font-weight: bold;"),
                 tag.Tag(
-                    _(mailingpreferences.status),
-                    tag_color=map_tag_color(mailingpreferences.status),
+                    _(subscription.status),
+                    tag_color=map_tag_color(subscription.status),
                     onclick="return false;",
                     style="margin-left: 1rem;",
                 ),
                 hg.SPAN(
-                    mailingpreferences.get_language_display(),
+                    subscription.get_language_display(),
                     style="padding-left:0.5rem;",
                 ),
                 style="margin-bottom: 1.5rem;",
             ),
         ),
         *interests,
+        R(
+            C(_("Last synchronized"), width=6, breakpoint="lg"),
+            C(
+                localize(
+                    localtime(subscription.latest_sync.sync_completed_datetime),
+                    use_l10n=settings.USE_L10N,
+                )
```

Comment:
Does ObjectFieldValue not cover this kind of formatting?

Reply:
It would :D
But this is all at render-time. I could rewrite the whole _display_subscription part to use Lazy.
I can do that if you prefer it. Originally I preferred non-lazy because it's much easier to debug at render-time if it's not lazy. But now that it works I could refactor it.

Reply:
Sure, this is up to you.